modules/friendlytag.js

```diff
@@ -76,6 +76,49 @@ Twinkle.tag.callback = function friendlytagCallback() {
 				]
 			});
 
+			form.append({
+				type: 'input',
+				label: 'Quick filter: ',
+				name: 'quickfilter',
+				size: '30px',
+				event: function twinkletagquickfilter() {
+					var $form = $('#tagWorkArea');
+					// flush the DOM of all existing underline spans
+					$form.find('.search-hit').each(function(i,e) {
+						var label_element = e.parentElement;
+						// This would convert <label>Hello <span class=search-hit>wo</span>rld</label>
+						// to <label>Hello world</label> // eslint-disable-next-line no-self-assign
+						label_element.textContent = label_element.textContent;
```

Comment:
Figured out an optimisation. Instead of reconstructing the entire label element, it is sufficient to re-assign the `textContent`. So this may look like a most pointless line of code, and eslint is definitely not happy, but it does the trick. 

The alternative would be to do something like 
```
$(label_element).find('span').text(this.textContent);  // replace the span with a text node containing the span's textcontent
label_element.normalize();   // re-joins the adjacent text nodes
```
or even worse, what I had done before. Both are clearly more expensive.

With this, the thing works quite smoothly even on IE.

Reply:
@siddharthvp Maybe it's cheating a bit, but wouldn't this work?  Seems to on my end.
```suggestion
						label_element.innerHTML = label_element.textContent;
```

Reply:
Aha that is clever, but do we really need to let eslint become our master 🤔

Reply:
Thanks — I agree we don't need to let the automaton control us, but it's a good indication that something will be weird/confusing/etc.  Going with `label_element.innerHTML = label_element.textContent` actually makes a lot more sense to the reader, I think, since it's clear we're replacing any HTML with just text, where we probably could've had another comment explaining that `textContent = textContent` strips the spans.

Anyway, one simple line is much nicer!